cmd/webhook/main.go

```diff
@@ -1,29 +1,94 @@
 package main
 
 import (
-	"flag"
+	"crypto/tls"
+	"crypto/x509"
 	"fmt"
+	"io/ioutil"
 	"net"
 	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/openshift/csi-driver-shared-resource/pkg/webhook/csidriver"
+	"github.com/openshift/csi-driver-shared-resource/pkg/webhook/dispatcher"
+	"github.com/spf13/cobra"
 )
 
 const (
 	WebhookName string = "sharedresourcecsidriver-csidriver"
 )
 
 var (
-	listenAddress = flag.String("listen", "0.0.0.0", "listen address")
-	listenPort    = flag.String("port", "5000", "port to listen on")
+	tlsCert       string
+	tlsKey        string
+	caCert        string
+	listenAddress string
+	listenPort    int
+	testHooks     bool
 )
 
-func index(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "<h1>The CSI Shared Resource webhook is under development<h1>")
+var CmdWebhook = &cobra.Command{
+	Use:     "csi-driver-shared-resource-webhook",
+	Version: "0.0.1",
+	Short:   "",
+	Long:    ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		startServer()
+	},
 }
-func main() {
-	flag.Parse()
-	http.HandleFunc("/"+WebhookName, index)
+
+func init() {
+	CmdWebhook.Flags().StringVar(&tlsCert, "tlscert", "",
```

Comment:
We don't really need to wrap all of these lines, they are not very long.

Reply:
I would like to keep those lines in the init function as we did here https://github.com/openshift/csi-driver-shared-resource/blob/master/cmd/csidriver/main.go#L132

Reply:
I meant you shouldn't split the line of code onto two lines :)